prep-css-flexbox/display-flex.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    div {
+      display: flex;
+    }
+
+    .card-container {
+      padding: 10px;
+    }
+
+    .card-deck {
+      padding: 10px;
+    }
```

Comment:
This rule-set should have a selector for `#card-deck` (the element id) instead of `.card-deck` (the element class).